keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -724,49 +662,40 @@ providers such as Okta, Azure AD, etc:
 
 #### Beta
 
-- Gather feedback
 - Complete benchmarks
-- Add metrics
-- Initial e2e test with an external provider completed and enabled
+- Support > 1 JWT authenticator and > 1 audiences
+- Enable automatic reload of the configuration
 
 #### GA
 
+- Gather feedback
 - Add a full documentation with examples for the most popular providers, e.g., Okta, Dex, Auth0
 - Migration guide
 - Deprecation warnings for non-structured OIDC provider configuration
+- Initial e2e test with an external provider completed and enabled
 
```

Comment:
From the CEL section:

> There will be a maximum allowed CEL expression cost per authenticator (no limit on total authenticators is required due to the issuer uniqueness requirement).

I think we need to update this with what we have today.

> There will be a upper bound of 5s for the CEL expression evaluation.

Did we implement this?